### DISCUSSION TITLE
cannot get into codespaces

### DISCUSSION THREAD

**Discussion by baltazargato** [community/community#28485] [Codespaces]
I am doing cs50 and vs code has worked perfectly until now. It has started to present a fail to fetch error whenever I try to enter vs code. When i try to download vs code and sign into github there, it does not let me sign in.

**Comment by jfullstackdev** [comment]
it's a known issue as you can see here, there are several posts, you may want to look at those posts

https://github.com/orgs/community/discussions?discussions_q=cs50
